README.md

```diff
@@ -2,18 +2,23 @@
 
 Welcome to AIBrix, an open-source initiative designed to provide essential building blocks to construct scalable GenAI inference infrastructure. AIBrix delivers a cloud-native solution optimized for deploying, managing, and scaling large language model (LLM) inference, tailored specifically to enterprise needs.
 
+
+<p align="center">
+| <a href="https://aibrix.readthedocs.io/latest/"><b>Documentation</b></a> | <a href="https://aibrix.github.io/"><b>Blog</b></a> | <a href="https://github.com/vllm-project/aibrix/blob/main/docs/paper/AIBrix_White_Paper_0219_2025.pdf"><b>Paper</b></a> | <a href="https://x.com/vllm_project"><b>Twitter/X</b></a> | <a href="https://slack.vllm.ai"><b>Developer Slack</b></a> |
```

Comment:
can we explicitly mention it's white paper now?

Reply:
Please use this slack link "https://vllm-dev.slack.com/archives/C08EQ883CSV" and it will redirect to right channel.

Reply:
Twitter/X is the vLLM general, is that ok?

Reply:
> can we explicitly mention it's white paper now?

updated to show "White Paper" instead of Paper

> Please use this slack link "https://vllm-dev.slack.com/archives/C08EQ883CSV" and it will redirect to right channel.
Updated

> Twitter/X is the vLLM general, is that ok?
It is okay.